### PR TITLE
[Docs] Recommendations for configuring KESL (Kaspersky Endpoint Security for Linux)

### PR DESCRIPTION
**Pull request by khasanovrs**
Added recommendations for configuring KESL (Kaspersky Endpoint Security for Linux)

Closes: #9556

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by kirkonru — APPROVED** [review]

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 1]
```diff
@@ -4,8 +4,244 @@ permalink: ru/security_software_setup.html
...
+kesl-control --set-settings 1 --add-exclusion /var/lib/containerd
+kesl-control --set-settings 1 --add-exclusion /var/lib/deckhouse
+kesl-control --set-settings 1 --add-exclusion /var/lib/etcd
+kesl-control --set-settings 1 --add-exclusion /var/lib/kubelet
+kesl-control --set-settings 1 --add-exclusion /var/lib/upmeter
+kesl-control --set-settings 1 --add-exclusion /var/log/containers
+kesl-control --set-settings 1 --add-exclusion /var/log/pods
+```
+
+_Примечание_: при добавлении может быть показано уведомление, что некоторых директорий не существует. Правило при этом будет добавлено — это нормально.
```
[the remark this answers is not in the document]
Comment (z9r5) [reply docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:60]: @Zhbert

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 2]
```diff
@@ -4,8 +4,244 @@ permalink: ru/security_software_setup.html
...
+kesl-control --set-settings 11 --add-exclusion /var/lib/upmeter
+kesl-control --set-settings 11 --add-exclusion /var/log/containers
+kesl-control --set-settings 11 --add-exclusion /var/log/pods
+```
+
+_Примечание_: при добавлении может быть показано уведомление, что некоторых директорий не существует. Правило при этом будет добавлено — это нормально.
+
+### Задача №12 «Управление сетевым экраном» (Firewall_Management)
+
+> Задачу **НЕОБХОДИМО ОТКЛЮЧИТЬ И НЕ ВКЛЮЧАТЬ**!!! Она приведёт к неработоспособности Deckhouse!
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:156]: @Zhbert тут level="warning"

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 3]
```diff
@@ -4,8 +4,244 @@ permalink: ru/security_software_setup.html
...
+
+Если задача включена и обнаружено её негативное влияние на Deckhouse, отключите задачу, выполнив команду:
+
+```shell
+kesl-control --stop-task 17
+```
+
+### Задача №20 «Анализ поведения» (Behavior_Detection)
+
+С настройками по умолчанию негативного влияния на работоспособность Deckhouse не оказывает.
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:215]:
```suggestion
С настройками по умолчанию, задача негативного влияния на работоспособность Deckhouse не оказывает.
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 4]
```diff
@@ -4,8 +4,244 @@ permalink: ru/security_software_setup.html
...
+
+Если задача включена и обнаружено её негативное влияние на Deckhouse, отключите задачу, выполнив команду:
+
+```shell
+kesl-control --stop-task 20
+```
+
+### Задача №21 «Контроль программ» (Application_Control)
+
+С настройками по умолчанию негативного влияния на работоспособность Deckhouse не оказывает.
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:227]:
```suggestion
С настройками по умолчанию, задача негативного влияния на работоспособность Deckhouse не оказывает.
```

File: docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md [hunk 5]
```diff
@@ -4,8 +4,244 @@ permalink: ru/security_software_setup.html
...
+
+Если задача включена и обнаружено её негативное влияние на Deckhouse, отключите задачу, выполнив команду:
+
+```shell
+kesl-control --stop-task 21
+```
+
+### Задача №22 «Веб-контроль» (Web_Control)
+
+Рекомендуется отключить.
```
Comment (z9r5) [inline docs/documentation/pages/SECURITY_SOFTWARE_SETUP_RU.md:239]:
```suggestion
Задачу рекомендуется отключить.
```

**Comment by deckhouse-BOaTswain** [comment]
Failure: cherry-pick commit d4871b7ce71e9d8916b9f5ccd3c7d0d4965db320 to the branch [release-1.63](https://github.com/deckhouse/deckhouse/tree/release-1.63) failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/10736765261) for details.

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9638](https://github.com/deckhouse/deckhouse/pull/9638) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Comment by z9r5** [comment]
Cherry-picked into release-1.63

**Pull request merged**